generators/wp/templates/chisel-starter-theme/Chisel/Media.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Chisel;
+
+/**
+ * Class Media
+ * @package Chisel
+ *
+ * Default media settings for Chisel
+ */
+class Media {
+
+	public function __construct() {
+		$this->addImagesSizes();
+		// add_action( 'after_setup_theme', array( $this, 'defaultMediaSetting' ) );
+		// add_filter( 'image_size_names_choose', array( $this, 'customImageSizes' ) );
+		add_action( 'jpeg_quality', array( $this, 'customJpegQuality') );
+		add_filter( 'embed_oembed_html', array( $this, 'customOembedFilter' ), 10, 4);
+	}
+
+	/**
+	 * Use this method to register custom image sizes
+	 */
+	public function addImagesSizes() {
+		// add_image_size( 'small', 225, 9999 );
+		// add_image_size( 'hero', 1600, 9999 );
+	}
+
+	/**
+	 * Add custom image sizes option to WP admin
+	 * @param  array $sizes Default sizes
+	 * @return array        Updated sizes
+	 */
+	public function customImageSizes( $sizes ) {
+		return array_merge( $sizes, array(
+			'small' => __( 'Small' ),
+		));
+	}
+
+	/**
+	 * Default settings when adding or editing post images
+	 */
+	public function defaultMediaSetting() {
+		update_option( 'image_default_align', 'center' );
+		update_option( 'image_default_link_type', 'none' );
+		update_option( 'image_default_size', 'full' );
+	}
+
+	/**
+	 * Sets custom JPG quality when resizing images
+	 * @return number JPG Quality
+	 */
+	public function customJpegQuality() {
+		return 80;
+	}
+
+	/**
+	 * Custom container for content videos
+	 */
+	function customOembedFilter( $html, $url, $attr, $post_ID ) {
+		return '<div class="c-video">' . $html . '</div>';
```

Comment:
WP Embed supports [much more than video](https://codex.wordpress.org/Embeds#Okay.2C_So_What_Sites_Can_I_Embed_From.3F), please consider changing class name.

Reply:
I've tried to make it more versatile plus improve performance, found a reference to it here https://developer.wordpress.org/reference/hooks/oembed_result/#user-contributed-notes

Small downsize could be that now embeds are cached and not parsed on each page load so it means if someones wants to make changes to this filter return code, it wouldn't change in already save embeds. Not sure if this is a real problem though.

Reply:
Could you add a comment about cache to the code? Feel free to merge after that.